tests/unit/state/test_particle.py

```diff
@@ -58,3 +58,15 @@ def test_create_antiparticle(
     assert anti_particle.width == comparison_particle.width
     assert anti_particle.state == comparison_particle.state
     assert anti_particle.name == "anti-" + particle_name
+
+
+def test_create_antiparticle_tilde():
+    anti_particles = DATABASE.find_subset("~")
+    assert len(anti_particles) == 9
+    for anti_particle in anti_particles.values():
+        particle_name = anti_particle.name.replace("~", "")
+        particle = DATABASE[particle_name]
+        created_anti_particle = create_antiparticle(
+            particle, anti_particle.name
+        )
+        assert created_anti_particle == anti_particle
```

Comment:
I meant this

```python
particle_name = anti_particle.name.replace("~", "")
created_anti_particle = create_antiparticle(
    anti_particle, particle_name
)
assert created_anti_particle == DATABASE[particle_name]
```

Reply:
Ok b323285ae9e896843a956e810be72120c54b9661